### PR TITLE
units validaton

### PR DESCRIPTION
**Pull request by ornemarioni**
This issue fixes units validation with astropy units
It is not finished yet, It is a draft version

### CODE REVIEW

File: galaxychop/galaxychop.py [hunk 1]
```diff
@@ -65,52 +81,94 @@ class Galaxy:
...
+    Etot_s = attr.ib(default=None)
+    Etot_g = attr.ib(default=None)
 
     components_s = attr.ib(default=None)
     components_g = attr.ib(default=None)
     metadata = attr.ib(default=None)
 
-    def energy(self, eps=0):
+    def __change_units_to_array__(f):
+        def new_method(self, *args, **kwargs):
```
Comment (vcristiani) [inline galaxychop/galaxychop.py:124]: Should the name of the method refer to what it does?

**Review by vcristiani — COMMENTED** [review]
All looks good. We should decorate the following methods with the same as in the energy method?

File: galaxychop/galaxychop.py (hunk 1 above)
Comment (ornemarioni) [reply galaxychop/galaxychop.py:124]: I don't really know. I only chose the name to know quickly what the decorator does.

**Comment by ornemarioni** [comment]
> All looks good. We should decorate the following methods with the same as in the energy method?

Yes. All the new methods must be decorated with this

**Comment by ornemarioni** [comment]
This fixes issue #46

**Pull request merged**